### DISCUSSION TITLE
for linux support can we have vulkan support or dxvk-native(non proton) support?

### DISCUSSION THREAD

**Discussion by leonpano2006** [piotrulos/MSCModLoader#184] [General]
can we run msc under vulkan(native) or dxvk-native vulkan under linux experimental support?
dxvk native should be possible since msc supports dx11

Note: dxvk native is [dxvk-native-2.3-steamrt-sniper.tar.gz](https://github.com/doitsujin/dxvk/releases/download/v2.3/dxvk-native-2.3-steamrt-sniper.tar.gz)
form [Releases](https://github.com/doitsujin/dxvk/releases)

**Comment by piotrulos** [comment]
But "linux experimental" as native linux build it doesn't use dx11, it uses openGL.

**Reply by leonpano2006** [reply]
i mean original game supports
but is vulkan possible?
or dxvk native (like what source game on linux uses)

**Reply by piotrulos** [reply]
dxvk works by only using proton 
Unity 5.0 is too early to have native vulkan support.

**Reply by leonpano2006** [reply]
check this [Joshua-Ashton/dxvk-native](https://github.com/Joshua-Ashton/dxvk-native)
but if you ask why is archived? 
i think is merged to [doitsujin/dxvk](https://github.com/doitsujin/dxvk)

and this files [dxvk-native-2.3-steamrt-sniper.tar.gz](https://github.com/doitsujin/dxvk/releases/download/v2.3/dxvk-native-2.3-steamrt-sniper.tar.gz) are linux binarys, not windows dll

**Reply by piotrulos** [reply]
As I said unity linux binaries are not compiled on directx, they are opengl, 
dxvk only translates directx calls it doesn't translate opengl calls.

**Reply by leonpano2006** [reply]
> As I said unity linux binaries are not compiled on directx, they are opengl, 
> 
> dxvk only translates directx calls it doesn't translate opengl calls.

So there is only zink left
Zink translate OpenGL to vulkan
